contrib/CMakeLists.txt

```diff
@@ -92,6 +92,10 @@ add_contrib (openldap-cmake openldap)
 add_contrib (grpc-cmake grpc)
 add_contrib (msgpack-c-cmake msgpack-c)
 
+#if (ENABLE_RUST)
```

Comment:
#if (ENABLE_RUST) is not necessary

Reply:
Flag ENABLE_RUST is used to disable Rust compilation completely. Corrosion is not needed if Rust is disabled, so we exclude it from build.